net-irc/irssi-xmpp/irssi-xmpp-0.54.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit toolchain-funcs
+
+DESCRIPTION="An irssi plugin providing Jabber/XMPP support"
+HOMEPAGE="https://cybione.org/~irssi-xmpp/"
+SRC_URI="https://github.com/cdidier/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+DEPEND="
+	>=net-irc/irssi-1.0.0
+	>=net-libs/loudmouth-1.4.0"
+
+RDEPEND="${DEPEND}"
+
+PATCHES=( "${FILESDIR}/${P}-irssi-1.0.patch" )
+
+src_prepare() {
+	default
+	sed -e "s/{MAKE} doc-install/{MAKE}/" \
+		-i Makefile || die #322355
+}
+
+src_compile() {
+	emake PREFIX=/usr CC="$(tc-getCC)"
```

Comment:
... which then makes me wonder if this is still needed?

Reply:
It works with this CC, I will test with the standard function

Reply:
We have to keep at least PREFIX because it defaults to `/usr/local` and then we can’t include `irssi-config.h`:
```
2020-04-26 21:05:55,543 - ae7a5d (stdout): compilation terminated.                                                                                                                   
2020-04-26 21:05:55,545 - ae7a5d (stdout): make[2]: *** [../rules.mk:10: xmpp-servers.o] Error 1                                                                                     
2020-04-26 21:05:55,545 - ae7a5d (stdout): make[2]: Leaving directory '/var/tmp/portage/net-irc/irssi-xmpp-0.54/work/irssi-xmpp-0.54/src/core'                                       
2020-04-26 21:05:55,546 - ae7a5d (stdout): make[1]: *** [Makefile:3: all] Error 2                                                                                                    
2020-04-26 21:05:55,546 - ae7a5d (stdout): make[1]: Leaving directory '/var/tmp/portage/net-irc/irssi-xmpp-0.54/work/irssi-xmpp-0.54/src'                                            
2020-04-26 21:05:55,546 - ae7a5d (stdout): make: *** [Makefile:4: all] Error 2                                                                                                       
2020-04-26 21:05:55,551 - ae7a5d (stdout):  * ERROR: net-irc/irssi-xmpp-0.54::gentoo failed (compile phase):                                                                         
2020-04-26 21:05:55,554 - ae7a5d (stdout):  *   emake failed                                                                                                                         
```